### ISSUE TITLE
prefers-reduced-motion

### ISSUE THREAD

**Issue by J-Kallunki** [vrk-kpa/suomifi-ui-components#247]
## The Problem

Respect if user client has `prefers-reduced-motion` set

## Proposed Solution

All motion should be utilized so that if `prefers-reduced-motion` is set the motion would be dissmissed or minimal if needed for information.

https://mobile.twitter.com/joshwcomeau/status/1189928574398222339?s=21
